cooperatives/x-road.md

```diff
@@ -19,3 +19,7 @@ parent: Cooperatives
 “X-Road is a free and open-source data exchange layer solution that enables organizations to exchange information securely over the Internet. [...] The basic idea of X-Road is that information systems do not exchange data directly with each other. Instead, information systems are connected through additional, standardized access points (Security Server) that implement the same technical specifications and therefore, can communicate with each other.” It was created by the Estonian government (who contracted it out to private vendors), but there was later a Finnish/Estonian MOU — signed by the PMs — governing collaboration on the project. Collaboration is now done via the Nordic Institute for Interoperability Solutions, which has both countries as members, as well as Iceland. It is now also in use by The Faroe Islands and El Salvador.
 
 "X-Road is a digital public good verified by the Digital Public Goods Alliance, and it’s released under the MIT open source license and is available free of charge."
+
+Officially, it is a non-profit association with three members: Estonia, Finland and Iceland. Therefore, when it comes to all of its operations, it is not a governmental agency. However, all of its funding is public. Funding comes directly from the Ministries of its different member countries. Namely, from the Finnish Ministry of Finance, the Estonian Ministry of Economic Affairs and Communications and the Icelandic Ministry of Finance and Economic Affairs.
+
+It was created by the Estonian government (who contracted it out to private vendors), but there was later a Finnish/Estonian MOU – signed by the Prime Ministers – governing collaboration on the project. Collaboration is now done via the Nordic Institute for Interoperability Solutions, which has both countries as members, as well as Iceland. It is now also in use by The Faroe Islands and El Salvador. 
```

Comment:
Although this gets into the distinction between governing members and users (conversation happening on slack), which we may want to represent differently in the table above. Let's resolve this in that conversation and figure out how we want to proceed, if these two countries are users but not governing members.

Reply:
On their website, it looks like it also lists around 30 other countries that are users. Because Iceland seems to be the only country of the 3 that seems to be a true member of the collaborative, I think we could probably leave it?

Reply:
Okay thanks!